MMUL-BASH-01/Exercises/Day-3/Exercise.004

```diff
@@ -0,0 +1,16 @@
+# Exercise 004 - Using string variables
+
+1) Log into the machine with the credentials you own;
+
+2) Touch a file called 'user_info.sh', append a bash shebang to it and make it
```

Comment:
Maybe "append the interpreter (shebang) at the beginning" is better

Reply:
fixed, shebang in parenthesis